app/views/users/show.html.erb

```diff
@@ -1,9 +1,36 @@
-<p id="notice"><%= notice %></p>
+<%= render  "shared/notice" %>
 
-<p>
-  <strong>Username:</strong>
-  <%= @user.username %>
-</p>
-
-<%= link_to 'Edit', edit_user_path(@user) %> |
-<%= link_to 'Back', users_path %>
+<div class="mt-5 pt-4 row mx-0 px-0 justify-content-center m-auto w-100 container">
+    <div class="upcoming eventz row d-flex flex-column m-auto col-12 w-100">
+        <h2 class="mb-5 text-center">Created events</h2>
+        <div class="row d-flex container justify-content-around">
+            <% @user.events.where(user_id: @user.id).each do |event| %>
+            <div class="card mb-4 col-md-3 mx-1 shadow " style="max-width: 260px; max-height: 300px;">
+                <div class="card-body">
+                    <h4 class="card-title mb-3"><%= event.name%></h4>
+                    <h6 class="card-subtitle mb-2 text-muted"><%=event.event_date%></h6>
+                    <p class="card-text"><%=truncate(event.description, length: 150)%></p>
+                    <%= link_to  "Learn more", event_path(event), class: "btn btn-block btn-outline-primary " %>
+                </div>
+            </div>
+            <% end %>
+        </div>
+    </div>
+</div>
+<div class="mt-5 pt-4 row mx-0 px-0 justify-content-center m-auto w-100 container">
+    <div class="upcoming eventz row d-flex flex-column m-auto col-12 w-100">
+        <h2 class="mb-5 text-center">Attended events</h2>
+        <div class="row d-flex container justify-content-around">
+            <% @user.event_attended.each do |event| %>
```

Comment:
Such a call must not be done in the view but in the controllers.
Create in the controller a view variable that will keep that result in order to be used in the views